tests/UnifiedSpecTests/Constraint/IsBsonTypeTest.php

```diff
@@ -163,15 +163,15 @@ private function assertResult($expected, Constraint $constraint, $value, string
 // phpcs:disable Squiz.Classes.ClassFileName.NoMatch
 class SerializableArray implements Serializable
 {
-    public function bsonSerialize()
+    public function bsonSerialize(): array
```

Comment:
Any reason these implementations don't include `#[ReturnTypeWillChange]`?

Reply:
Since these are test classes where we're not bound by BC constraints, we can add the type right away.